src/main/java/doore/login/api/LoginController.java

```diff
@@ -0,0 +1,24 @@
+package doore.login.api;
+
+import doore.login.application.LoginService;
+import doore.login.application.dto.request.GoogleLoginRequest;
+import doore.login.application.dto.response.LoginResponse;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/login")
+@RequiredArgsConstructor
+public class LoginController {
```

Comment:
```suggestion
@RestController
@RequiredArgsConstructor
@RequestMapping("/login")
public class LoginController {
```
좀 Nit긴 한데, 매핑 url 어노테이션을 바로 위에 붙여주는 건 어떨까요?
+) 컨트롤러 단에도 `@Validated`를 붙여줘야 `@Valid`가 동작하는 것으로 알고 있습니다!

Reply:
반영하겠습니다!